src/ralph/security/models.py

```diff
@@ -46,7 +46,12 @@ class Vulnerability(
     name = models.CharField(
         verbose_name=_("name"),
         max_length=1024,
-        unique=False)
+        unique=False
+    )
+    short_name = models.CharField(
+        verbose_name=_("short name"),
+        max_length=1024
```

Comment:
From what I see unique is False by default: https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L135

Reply:
I renamed this field to ``display_name``